stumpy/scrump.py

```diff
@@ -22,8 +22,11 @@ def _get_max_order_idx(m, n, orders, start, percentage):
     m : int
         Window size
 
-    n : int
-        Matrix profile length
+    n_A : int
+        Length of the first time series
+
+    n_B : int
+        Lenght of the second time series
```

Comment:
"Length of time series T_B"

Reply:
Should only one of these be necessary (i.e., should one of `n_A` or `n_B` be optional) if we are doing a self-join?